modules/s3-bucket/remote-state.tf

```
@@ -5,7 +5,7 @@ module "account_map" {
   component   = "account-map"
   environment = var.account_map_environment_name
   stage       = var.account_map_stage_name
-  tenant      = coalesce(var.account_map_tenant_name, module.this.tenant)
+  tenant      = try(coalesce(var.account_map_tenant_name, module.this.tenant), "")
```

Would this work instead since `module.this.tenant` is already provided by the `context` ?

```suggestion
  tenant      = var.account_map_tenant_name
```

---

@nitrocode I think it probably would. @joe-niland you mind testing this change out to confirm?

---

@nitrocode yes that does work, thanks.

I actually wasn't aware of [this behaviour](https://github.com/cloudposse/terraform-null-label/blob/master/main.tf#L51)!

I've modified the PR